### ISSUE TITLE
Can you share the pretrained checkpoints?

### ISSUE THREAD

**Issue by chufall** [ozerlabs-proxy/VisIrNet#39]
Hi,
May I ask you to share the pretrained checkpoints? such as "featureEmbeddingBackBone-5827618d1f2048e0819760780721fcbc-7.keras"
in your notebook.

Thanks a lot
Q.C.

**Comment by ysh5858** [comment]
Hello, may I ask what the content and format of the label file are in the dataset?

**Comment by jonathanmcgillivray** [comment]
Thanks for the paper and code repo. I would also find the model weights used to get the results in the paper useful if you are able to share them.

@ysh5858 take a look at the dataset format in this repo - https://github.com/placeforyiming/CVPR21-Deep-Lucas-Kanade-Homography. I think it's the same format.
